File: 0001 Code Chef/0019 Water Consumption/implementation/0006 php/water_consumption.php
<?php
    $test = (int) fgets(STDIN);
    while($test > 0){
        $water = (int) fgets(STDIN);
        if($water >= 2000){
            echo "YES\n";
        }else{
            echo "NO\n";
        }
        $test = $test - 1;
    }
?>

File: 0001 Code Chef/0019 Water Consumption/implementation/0009 dart/water_consumption.dart
import 'dart:io';

void main(){
  String? stringInput = stdin.readLineSync();
  if(stringInput != null){
    int? test = int.tryParse(stringInput);
    if(test != null){
      int count = test;
      while(count > 0){
        String? stringInput = stdin.readLineSync();
        if(stringInput != null){
          int? water = int.tryParse(stringInput);
          if(water != null){
            if(water >= 2000){
              print("YES");
            }else{
              print("NO");
            }
          }
        }
        count = count - 1;
      }
    }
  }
}